src/steamship/agents/memory/chathistory.py

```diff
@@ -160,3 +163,29 @@ def messages(self) -> List[Block]:
     @property
     def client(self) -> Client:
         return self.file.client
+
+    def messages_as_string(
+        self,
+        memory_window_strategy: MessageSelector,
```

Comment:
we should update the param name too...
```suggestion
        message_selector: MessageSelector,
```